benchmarks/backgrounds/Snakefile

```diff
@@ -0,0 +1,107 @@
+import os
+import shutil
+
+from snakemake.remote.S3 import RemoteProvider as S3RemoteProvider
+
+
+S3 = S3RemoteProvider(
+    endpoint_url="https://eics3.sdcc.bnl.gov:9000",
+    access_key_id=os.environ["S3_ACCESS_KEY"],
+    secret_access_key=os.environ["S3_SECRET_KEY"],
+)
+
+
+rule backgrounds_get_beam_gas_electron:
+    input:
+        S3.remote("eictest/EPIC/EVGEN/BACKGROUNDS/BEAMGAS/electron/beam_gas_ep_10GeV_foam_emin10keV_10Mevt_vtx.hepmc"),
+    output:
+        "input/backgrounds/beam_gas_electron.hepmc",
+    run:
+        shutil.move(input[0], output[0])
+
+
+rule backgrounds_get_beam_gas_proton:
+    input:
+        S3.remote("eictest/EPIC/EVGEN/BACKGROUNDS/BEAMGAS/proton/ProtonBeamGasEvents/100GeV/100GeV_1.hepmc"),
+    output:
+        "input/backgrounds/beam_gas_proton.hepmc",
+    run:
+        shutil.move(input[0], output[0])
+
+
+rule backgrounds_get_DIS:
+    input:
+        S3.remote("eictest/EPIC/FULL/23.11.0/epic_craterlake/DIS/NC/10x100/minQ2=10/pythia8NCDIS_10x100_minQ2=10_beamEffects_xAngle=-0.025_hiDiv_1.0000.edm4hep.root"),
```

Comment:
Don't we want a pythia6 lowQ2 background file here?

Reply:
There are two things here:
1. With this PR I wanted to add code that was made for the original talk 
(And actually, I now realize, the minQ2=1 was used originally. I don't understand why minQ2=10 is used here)
2. lowQ2 sample (0<Q^2<1) S3/eictest/EPIC/EVGEN/SIDIS/pythia6-eic/1.0.0/10x100/q2_0to1/ is unusable until I manage to figure out which sample fills the rest of Q^2 spectrum, what cross sections are to combine those, etc. (I'm eyeing the S3/eictest/EPIC/EVGEN/SIDIS/pythia6/ep_10x100/hepmc_ip6/noradcor/ for that)

Reply:
```suggestion
        S3.remote("eictest/EPIC/FULL/23.11.0/epic_craterlake/DIS/NC/10x100/minQ2=1/pythia8NCDIS_10x100_minQ2=1_beamEffects_xAngle=-0.025_hiDiv_1.0000.edm4hep.root"),
```